### ISSUE TITLE
Compilation failure

### ISSUE THREAD

**Issue by CelineBarrier** [ichthyop/ichthyop#34]
Hello, 
I installed a jdk and maven on a server but when I run Ichthyop installation, I have the following error:

[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project ichthyop: Compilation failure: Compilation failure:
[ERROR] /home/fbosseur/ichthyop-master/src/main/java/org/previmer/ichthyop/ui/logging/JTextAreaOutputStream.java:[88,58] cannot find symbol
[ERROR]   symbol:   method modelToView2D(int)
[ERROR]   location: variable m_textArea of type javax.swing.JTextArea
[ERROR] /home/fbosseur/ichthyop-master/src/main/java/org/previmer/ichthyop/ui/logging/JTextAreaOutputStream.java:[105,54] cannot find symbol
[ERROR]   symbol:   method modelToView2D(int)
[ERROR]   location: variable m_textArea of type javax.swing.JTextArea

Do you know where is the problem ?

Thank you, 

Céline

**Comment by barriern** [comment]
Hi Celine. Which version of Java do you use?

**Comment by CelineBarrier** [comment]
Thank you for your reply,
here is the output of the commands:

[fbosseur@romeologin1 ~]$ javac -version
javac 1.8.0_202

[fbosseur@romeologin1 ~]$ mvn -v
Apache Maven 3.8.4 (9b656c72d54e5bacbed989b64718c159fe39b537)
Maven home: /home/fbosseur/maven/apache-maven-3.8.4
Java version: 1.8.0_202, vendor: Oracle Corporation, runtime: /home/fbosseur/java/jdk1.8.0_202/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "3.10.0-1160.2.2.el7.x86_64", arch: "amd64", family: "unix"

**Comment by barriern** [comment]
And what is the result of `which java`? 

I suspect that it is something related to an outdated version of Java. 

Could you try to install an updated version of Java?

On Linux you can do it as follows:

```
sudo apt install openjdk-8-jdk
```

**Comment by barriern** [comment]
Hi Celine.

Did you manage to compile the code?

**Comment by CelineBarrier** [comment]
Yes, sorry I forgot to reply, my intern had a wrong version of java

Le ven. 11 févr. 2022 à 08:46, Nicolas Barrier ***@***.***> a
écrit :

**Comment by barriern** [comment]
Ok thanks, I close the issue then
